default_app/index.html

```diff
@@ -53,18 +74,22 @@
     }
 
     #holder {
-      border: 2px dashed #448691;
       margin: 0 auto;
-      height: 300px;
-      color: #45828E;
-      font-size: 40px;
-      line-height: 300px;
+      height: 275px;
+      border: 1px solid #e0e5e6;
```

Comment:
Should this be dashed to signify it's a special box? It currently has the same formatting as the command box.